### PR TITLE
Hvh faction

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Prep work for HvH gamemode. On the menu:

Rebel jobs with rebel access, rebel comms etc..

Still a lot to do to make this faction playable, but this is enough for one PR

Same that the other PR but without the ship, since it's no ready

## Changelog
:cl:
add: Rebel faction
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
> Is there a way you can add a varb for different announcement system for the hvh faction only in your future prs regarding this?

Have been discussed in discord. He has plans to do such.

**Comment by David-Stormwell** [comment]
I'll leave this statement here since I'm required to.
HvH was planned, but not this early as we need to adjust quite a few things beforehand.
Feel free to go ahead with this PR, as I'd like to see what you can come up with though.

File: code/__DEFINES/access.dm [hunk 1]
```diff
@@ -54,6 +86,7 @@ GLOBAL_LIST_EMPTY(all_req_one_access)
 #define ACCESS_ILLEGAL_PIRATE		201
 
 //Temporary until I turn these into defines/bitflags and develop proper IF tagging.
```
Comment (SplinterGP) [inline code/__DEFINES/access.dm:88]: "temporary"

Comment (BraveMole) [reply code/__DEFINES/access.dm:88]: 2 years latter ; still here

File: code/controllers/subsystem/job.dm [hunk 2]
```diff
@@ -49,7 +49,10 @@ SUBSYSTEM_DEF(job)
 		if(!job.map_check())
 			continue
 		occupations += job
-		name_occupations[job.title] = job
+		var/job_name = job.title
+		if(job.faction == FACTION_TERRAGOV_REBEL)
+			job_name += " Rebel"
+		name_occupations[job_name] = job
```
Comment (TiviPlus) [inline code/controllers/subsystem/job.dm:55]: why though just name it properly

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -2135,7 +2135,8 @@ Status: [status ? status : "Unknown"] | Damage: [health ? health : "None"]
 				for(var/path in job_paths)
 					var/datum/outfit/O = path
 					if(initial(O.can_be_admin_equipped))
-						job_outfits[initial(O.name)] = path
+						var/outfit_name = initial(O.name)
+						job_outfits[outfit_name] = path
```
Comment (TiviPlus) [inline code/modules/admin/topic.dm:2139]: uh?

File: code/controllers/subsystem/job.dm (hunk 2 above)
Comment (BraveMole) [reply code/controllers/subsystem/job.dm:55]: So admins can give it from VV.
It was mostly usefull from testing, but this might be usefull in game too

Comment (TiviPlus) [reply code/controllers/subsystem/job.dm:55]: give what from vv this is job names

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**